tests/_support/Page/User.php

```diff
@@ -2,7 +2,6 @@
 
 namespace Page;
 
-
 use Behat\Behat\Context\Context;
 use Behat\Gherkin\Node\TableNode;
 use PhpSpec\Exception\Example\PendingException;
```

Comment:
Codacy is complaining about short variables here but we actually want to use `$I` as it makes sense when reading the code like gherkin style. The way to disable the codacy test here is to add this before the class:

```
/**
 * @SuppressWarnings(PHPMD.ShortVariable)
 */
class User implements Context {
```